db/migrate/20171028223910_add_db_index_to_user_authentications.rb

```diff
@@ -0,0 +1,6 @@
+class AddDbIndexToUserAuthentications < ActiveRecord::Migration[4.2]
```

Comment:
@laertispappas please use `SpreeExtension::Migration[4.2]` so it will run smoothly on Rails 4.2, 5.0 and 5.1

Reply:
@damianlegawiec Ack. Thanks

Reply:
@damianlegawiec Done